plugins/infoblox/icon_infoblox/connection/connection.py

```diff
@@ -1,23 +1,26 @@
-import komand
-from .schema import ConnectionSchema
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema, Input
 
 # Custom imports below
 from icon_infoblox.util.infoblox import InfobloxConnection
 
 
-class Connection(komand.Connection):
+class Connection(insightconnect_plugin_runtime.Connection):
     def __init__(self):
         super(self.__class__, self).__init__(input=ConnectionSchema())
         self.infoblox_connection = None
 
     def connect(self, params):
         self.logger.info("Connect: Connecting...")
 
-        url = params.get("url")
-        api_version = params.get("api_version")
-        username = params.get("credentials").get("username")
-        password = params.get("credentials").get("password")
+        url = params.get(Input.URL)
+        api_version = params.get(Input.API_VERSION)
+        username = params.get(Input.CREDENTIALS).get("username")
+        password = params.get(Input.CREDENTIALS).get("password")
```

Comment:
To avoid `KeyError`
```suggestion
        username = params.get(Input.CREDENTIALS, {}).get("username")
        password = params.get(Input.CREDENTIALS), {}.get("password")
```